.env

```
@@ -1,7 +1,7 @@
 NEXT_PUBLIC_SUPABASE_URL=https://fcqqkxwlntnrtjfbcioz.supabase.co
 NEXT_PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImZjcXFreHdsbnRucnRqZmJjaW96Iiwicm9sZSI6ImFub24iLCJpYXQiOjE2ODM2NjUzNjYsImV4cCI6MTk5OTI0MTM2Nn0.RyjXi3kqR1P8fE-bZWmxxa2KmOkRp3WqJ5593-c6OWs
 NEXT_PUBLIC_GS_API_URL=https://beta.gs-api.opensauced.pizza/v1
-NEXT_PUBLIC_API_URL=https://beta.api.opensauced.pizza/v1
+NEXT_PUBLIC_API_URL=http://localhost:3001/v1
```

Revert this change. You can create a `.env.local` to store custom environment settings

```suggestion
NEXT_PUBLIC_API_URL=https://beta.api.opensauced.pizza/v1
```